docs/writing-tests/test-runner.md

```diff
@@ -238,6 +237,105 @@ When the test-runner executes, your existing tests will go through the following
+| --------- | ----------------------------------------------------------------------------- |
+| `exclude` | Prevents stories if they match the provided tags from being tested.           |
+| `include` | Defines a subset of stories only to be tested if they match the enabled tags. |
+| `skip`    | Skips testing on stories if they match the provided tags.                     |
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/test-runner-tags-config.js.mdx',
+    'common/test-runner-tags-config.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<Callout variant="info" icon="ℹ️">
+
+Running tests with the CLI flags takes precedence over the options provided in the configuration file and will override the available options in the configuration file.
+
+</Callout>
+
+#### Disabling tests
+
+If you want to prevent specific stories from being tested by the test-runner, you can configure your story with a custom tag, enable it to the test-runner configuration file or run the test-runner with the `--excludeTags` [CLI](#cli-options) flag and exclude them from testing. This is helpful when you want to exclude stories that are not yet ready for testing or are irrelevant to your tests. For example:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/my-component-exclude-tags.story.js.mdx',
+    'common/my-component-exclude-tags.story.ts.mdx',
+    'common/test-runner-tags-exclude.config.js.mdx',
+    'common/test-runner-tags-exclude.config.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+#### Run tests for a subset of stories
+
+To allow the test-runner only to run tests on a specific story or subset of stories, you can configure the story with a custom tag, enable it in the test-runner configuration file or run the test-runner with the `--includeTags` [CLI](#cli-options) flag and include them in your tests. For example, if you wanted to run tests based on the `test-only` tag, you can adjust your configuration as follows:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/my-component-include-tags.story.js.mdx',
+    'common/my-component-include-tags.story.ts.mdx',
+    'common/test-runner-tags-include.config.js.mdx',
+    'common/test-runner-tags-include.config.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<Callout variant="info" icon="ℹ️">
+
+Applying tags for the component's stories should either be done at the component level (using `meta`) or at the story level. Importing tags across stories is not supported in Storybook and won't work as intended.
+
+</Callout>
+
+#### Skip tests
+
+If you want to skip running tests on a particular story or subset of stories, you can configure your story with a custom tag, enable it in the test-runner configuration file, or run the test-runner with the `--skipTags` [CLI](#cli-options) flag. Running tests with this option will cause the test-runner to skip over the story and flag them accordingly in the test results. For example:
```

Comment:
Can you check this PR and maybe apply a similar wording?
https://github.com/storybookjs/test-runner/pull/396